src/Microsoft.DotNet.Darc/tests/Microsoft.DotNet.Darc.VirtualMonoRepo.E2E.Tests/VmrE2ETestsBase.cs

```diff
@@ -0,0 +1,274 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using CSharpx;
+using FluentAssertions;
+using Microsoft.DotNet.DarcLib.Helpers;
+using NUnit.Framework;
+
+#nullable enable
+namespace Microsoft.DotNet.Darc.Tests.VirtualMonoRepo;
+
+public abstract class VmrE2ETestsBase : GitOperations
```

Comment:
Instead of inheriting from `GitOperations`, it's better to create a class like `GitOperationsHelper` and then create a field in the `VmrE2ETestsBase` class:

```csharp
protected GitOperationsHelper GitOperations { get; } = new();
```

then you can call it from the tests:

```csharp
GitOperations.InitialCommit(/* ... */)
```